src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/package.json

```diff
@@ -15,6 +15,7 @@
     "typescript": "3.3.4000"
   },
   "dependencies": {
+    "ps-list": "^7.0.0",
```

Comment:
Just remembered, you'll also need to make sure this is tracked in component governance as well: https://dev.azure.com/dnceng/internal/_componentGovernance/dotnet-aspnetcore-tooling?_a=alerts&typeId=1981311&alerts-view-option=active